packages/integration-tests/exec/build_and_run.sh

```diff
@@ -15,7 +15,7 @@ if [ -n "$REBUILD" ]; then
 
   if [ -n "$FETCH_DEPS" ]; then
     echo -e "\nFetching dependencies (this will take forever the first time time)..."
-    yarn --cwd $ROOT_DIR --verbose
+    yarn --cwd $ROOT_DIR --verbose --frozen-lockfile
```

Comment:
The frozen lockfile flag works now that we've fixed that weird environment variable setting bug--that right?

Reply:
The `frozen-lockfile` flag prevents problems when updating the git repo to prevent side effects from the call to yarn which would otherwise update the `yarn.lock` file and then git would be unhappy when trying to pull.